### DISCUSSION TITLE
so-elastic-auth missing

### DISCUSSION THREAD

**Discussion by seg1b** [Security-Onion-Solutions/securityonion#3304] [Unsupported Versions] [answered]
Hello,

i've installed so 2.3.21 from the iso file. Everything works fine, but then i've tried to enable the elastic features. The command so-enable-features seems to work. But i don't have the command so-elastic-auth in the folder /usr/sbin. I used find, but the file is not part of my installation.
I've already tried to pick the file direct from the github project. After execution, following errors occur:

[root@sso sbin]# so-elastic-auth
/sbin/so-elastic-auth: line 18: /usr/sbin/so-elastic-common: No such file or directory
/sbin/so-elastic-auth: line 20: /etc/nsm/securityonion.conf: No such file or directory
Elasticsearch is not enabled!

Elasticsearch is working, i use Kibana to view Logs from different servers.

I've already updated to 2.3.30, the problem still exists.

What is my mistake, can anyone help me with this problem?

**Comment by dougburks** [comment, marked as answer]
It looks like you copied `so-elastic-auth` from the old Security Onion 16.04, so that's not expected to work on the new Security Onion 2.

We are currently working on improving our integration with Elastic auth, so stay tuned!

**Comment by TOoSmOotH** [comment]
https://github.com/Security-Onion-Solutions/securityonion/discussions/4434
